### PR TITLE
5.3.3 history

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Work in progress
Some entries are related to some PR still opened but closed to be in

# Testing this PR

Nothing to test

# Related reading
https://trello.com/c/duLrEnjy/33-version-history-pr

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+
+Sysadmin changes include:
+
+- clarified the upgrade of the "Open With" option
+- allowed installation of OMERO.web with ice 3.5
+- fixed recursive loading of feedback in OMERO.web
+
+Developer updates include:
+
+- added example on how to retrieved shapes from a ROI using batch querying for scalability
```
Comment (mtbc) [inline history.txt]: "retrieve"

File: history.txt [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
 
+5.3.3 (June 2017)
+-----------------
+
+This is a bug-fix release.
+
+Improvements include:
+
+- support for 2 new lookup tables from `Janelia <https://www.janelia.org/>`_
+- fixed loading of wells in right-hand panel of OMERO.web
```
Comment (will-moore) [inline history.txt]: Maybe "enabled the Preview tab for Wells in right-hand panel of OMERO.web"

File: history.txt [hunk 3]
```diff
@@ -1,6 +1,35 @@
...
+
+- added example on how to retrieved shapes from a ROI using batch querying for scalability
+- improved logging of error during deletion
+- added new methods to Java Gateway
+- improved login options in Java Gateway
+- specified image's dataset in url to give more context to OMERO.web apps 
+
+
+This release also upgrades the version of Bio-Formats which OMERO uses to
+`5.5.2 <https://www.openmicroscopy.org/site/support/bio-formats5.5/about/whats-new.html>`_.
```
Comment (will-moore) [inline history.txt]: Could link to the 5.5.2 section of that page https://www.openmicroscopy.org/site/support/bio-formats5.5/about/whats-new.html#june-15 (unless we know that there's not going to be any more 5.5.x releases added to that page?)

File: history.txt [hunk 4]
```diff
@@ -1,6 +1,35 @@
...
 =====================
 
+5.3.3 (June 2017)
+-----------------
+
+This is a bug-fix release.
+
+Improvements include:
+
+- support for 2 new lookup tables from `Janelia <https://www.janelia.org/>`_
```
Comment (hflynn) [inline history.txt]: two

File: history.txt [hunk 5]
```diff
@@ -1,6 +1,35 @@
...
+Sysadmin changes include:
+
+- clarified the upgrade of the "Open With" option
+- allowed installation of OMERO.web with ice 3.5
+- fixed recursive loading of feedback in OMERO.web
+
+Developer updates include:
+
+- added example on how to retrieve shapes from a ROI using batch querying for scalability
+- improved logging of error during deletion
```
Comment (hflynn) [inline history.txt]: errors

File: history.txt [hunk 6]
```diff
@@ -1,6 +1,35 @@
...
 
+5.3.3 (June 2017)
+-----------------
+
+This is a bug-fix release.
+
+Improvements include:
+
+- support for 2 new lookup tables from `Janelia <https://www.janelia.org/>`_
+- enabled the Preview tab for wells in right-hand panel of OMERO.web
```
Comment (hflynn) [inline history.txt]: We had "preview of wells now available in the right-hand panel" in 5.3.2

File: history.txt [hunk 7]
```diff
@@ -1,6 +1,35 @@
...
+
+Sysadmin changes include:
+
+- clarified the upgrade of the "Open With" option
+- allowed installation of OMERO.web with ice 3.5
+- fixed recursive loading of feedback in OMERO.web
+
+Developer updates include:
+
+- added example on how to retrieve shapes from a ROI using batch querying for scalability
```
Comment (hflynn) [inline history.txt]: an example or examples?
on -> of

File: history.txt [hunk 8]
```diff
@@ -1,6 +1,35 @@
...
+- improved logging of error during deletion
+- added new methods to Java Gateway
+- improved login options in Java Gateway
+- specified image's dataset in url to give more context to OMERO.web apps
```
Comment (hflynn) [inline history.txt]: "...an image's dataset in its URL..."

**Review by hflynn — COMMENTED** [review]
A few minor nitpicks and a clarification needed

**Comment by jburel** [comment]
@hflynn done

**Comment by hflynn** [comment]
You missed url -> URL @jburel but otherwise it looks good to me now

**Comment by jburel** [comment]
@hflynn I have added a new entry for the OpenSSL issue under sysadmin
After that I think we should be good to go

**Comment by will-moore** [comment]
Looks good.

File: history.txt [hunk 9]
```diff
@@ -1,6 +1,38 @@
...
+- support for two new lookup tables from `Janelia <https://www.janelia.org/>`_
+- fixed loading of Well in right-panel when browsing Well under Tag tree or from search results
+
+Sysadmin changes include:
+
+- clarified the upgrade of the "Open With" option
+- allowed installation of OMERO.web with ice 3.5
+- fixed recursive loading of feedback in OMERO.web
+- provided patch for OMERO.server installation on OS using 
+`OpenSSL 1.1.0 <https://www.openmicroscopy.org/site/support/omero5.3/sysadmins/troubleshooting.html>`_
```
Comment (hflynn) [inline history.txt]: We usually use internal doc links rather than treating them as external URLs

**Comment by jburel** [comment]
@hflynn done

**Comment by hflynn** [comment]
Is the link on a new line (sorry bit hard to tell)? If so, you'll need to indent it to have it be part of the bullet point or the sphinx build will complain when this is copied over.

Otherwise 👍

**Comment by hflynn** [comment]
Should be good to merge once green

**Comment by jburel** [comment]
@hflynn Could you take care of the pre-release announcement now that we are happy with the content?

**Comment by hflynn** [comment]
Will do

**Comment by jburel** [comment]
Thanks

**Comment by pwalczysko** [comment]
Failing travis ?

**Comment by pwalczysko** [comment]
Other than travis, reads well and ready to merge FMPOV.

**Comment by jburel** [comment]
yes travis is a real pain at the moment.

**Comment by jburel** [comment]
@hflynn sorry one more entry! This should be the last one

File: history.txt [hunk 10]
```diff
@@ -10,6 +10,7 @@ Improvements include:
 
 - support for two new lookup tables from `Janelia <https://www.janelia.org/>`_
 - fixed loading of Well in right-panel when browsing Well under Tag tree or from search results
+- fixed rotations of labels in figure scripts.
```
Comment (mtbc) [inline history.txt]: acquired an unusual terminal period

Comment (hflynn) [inline history.txt]: Unneeded full stop and rotation should be singular in this context

**Comment by jburel** [comment]
finally green

**Pull request merged**